content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+```text
+|- my-hugo-site/
+   |- themes/
+      |- my-hugo-theme/
+         |- layouts/
+         |- theme.toml
+         |- other files and folders
+   |- other files and folders
+```
 
- * **HTML files with any non-standard syntax are source files.** Some themes make use of `partials`, `shortcodes`, `internal templates` to create elements. Those must be compiled by `Hugo` to generate standard HTML. Those HTML files are located in `layouts` folder.
+#### Method 2: Install the theme as Git submodule
 
- * **exampleSite folder has source files.** This folder includes all `content`, `data`, and `config` files which must be compiled to run on the browser.
+If you are using a Git repository for your website and the theme is available in a Git repository then you can install the theme as submodule.
 
- * **Fonts, images, and plugins are static files.** These don't require any processing to be used and aren't generated from any other source. Fonts and Plugins are located in `static` folder. And Images are located in `exampleSite/static` folder because images need to change when you build a real site.
+```bash
+cd my-hugo-website
+git submodule add git@github.com:themeauthorname/my-hugo-theme.git themes/my-hugo-theme
+git commit -m "add theme my-hugo-theme as submodule"
+```
 
- * **The output of those above examples are compiled files.** The CSS files are generated by processing the source SCSS. The HTML files are generated by processing the pseudo-HTML source. And all pages are generated from exampleSite contents. All are generated from source files and will be overwritten if the source is compiled again. Output files can be found on `exampleSite/public` folder, when you run `hugo --themesDir ../..` this command. If you want the minified output, then run `hugo --minify --themesDir ../..`.
+This will create a subfolder in themes/my-hugo-theme that contains the files for your theme. 
 
----
+#### Method 3: Install the theme as Hugo module
 
-## Working with source files
+If the theme is available as a Hugo module you can install it as module. Hugo will then download the theme when it compiles or serves your website.
 
-Now that you've got your theme running and understand the compiling process, let's discuss the proper ways to edit and customize your theme. You can directly customize your template from the source file. For a better experience, open the whole theme folder with a text-editor or IDM (i.e. [Visual Studio Code](https://code.visualstudio.com/)).
+Initialize the use of modules in your website with the following command:
 
-### Site configuration
+```bash
+hugo mod init github.com/my-username/my-hugo-theme
+```
 
-You have a `config.toml` file in the `exampleSite` folder. Which gives you the ability to configure your site, such as `changing logo` or `customize menu`.
+After that you will find a file `go.mod` in your root directory. Add this file to your repository.
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)